backend/src/controllers/ProfessionalController.ts

```diff
@@ -73,9 +73,10 @@ export default class ProfessionalController {
   public static async login(req: Request, res: Response) {
     const { email, password } = req.body;
 
-    const professional = await professionalRepository.findByEmail(email);
+    if (!(await professionalRepository.existsEmail(email)))
+      return badRequest(res, 'Credenciais inválidas');
 
-    if (!professional) return badRequest(res, 'Credenciais inválidas');
+    const professional = await professionalRepository.findByEmail(email);
```

Comment:
Não é melhor buscar por completo e verificar se for nulo?

Reply:
é pq eu havia mudado o findByEmail do repository pra pegar tmb o phone e workplace do professional, dai se não existisse professional com o email iria dar erro, só que na verdade da pra adicionar no repository se não tiver o professional com email só retornar nulo mesmo, vou arrumar isso aí

Reply:
Resolvido no último commit